mne/tests/test_report.py

```diff
@@ -170,6 +174,43 @@ def test_render_report(renderer, tmpdir):
         report.add_figs_to_section(['foo'], 'caption', 'section')
 
 
+@testing.requires_testing_data
+def test_render_non_fiff(tmpdir):
+    """Test rendering non-FIFF files for mne report."""
+    tempdir = str(tmpdir)
+    fnames_in = [bdf_fname, edf_fname]
+    fnames_out = []
+    for fname in fnames_in:
+        basename = op.basename(fname)
+        basename, ext = op.splitext(basename)
+        fname_out = f'{basename}_raw{ext}'
+        outpath = op.join(tempdir, fname_out)
+        shutil.copyfile(fname, outpath)
+        fnames_out.append(fname_out)
+
+    report = Report()
+    report.parse_folder(data_path=tempdir, render_bem=False, on_error='raise')
+
+    # Check correct paths and filenames
+    for fname in fnames_out:
+        assert (op.basename(fname) in
+                [op.basename(x) for x in report.fnames])
+        assert (''.join(report.html).find(op.basename(fname)) != -1)
+
+    assert_equal(len(report.fnames), len(fnames_out))
```

Comment:
We prefer plain `assert` to `assert_equal` nowadays usually -- `assert_equal` is less informative than plain `assert len(report.fnames) == len(fnames_out)`. Feel free to remove all uses in this file if you want, but at least let's not add more of them

Reply:
I've replaced all calls to `assert_equal` with plain `assert` now